scylladb/sphinx-scylladb-theme
# feat: add new logo & mascots

Closes #1181 


![image](https://github.com/user-attachments/assets/c1e51153-10c8-4416-85e2-c9d74d6482a2)

## How to test

Build the docs and see the new logo.

To check the available mascots, open: http://127.0.0.1:5500/examples/mascots/

## Next steps

Once we release a new version of the theme, we will need to manually change the mascots in the following projects:

- https://docs.scylladb.com/stable/
- https://iot.scylladb.com/
- https://feature-store.scylladb.com/stable/
- https://cloud.docs.scylladb.com/stable/
- https://gaming-leaderboard.scylladb.com/stable/
